ansible/deploy_swarm_sip.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
nit: This playbook isn't specific to sip, or swarm. Could just be called container-infra.yml?

Reply:
Agree.